builder/bundles.go

```diff
@@ -480,7 +506,35 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func extractRpm(baseDir string, rpm string) error {
+	cmd := exec.Command("rpm2cpio", rpm)
+	cmd.Env = os.Environ()
+	var outBuf bytes.Buffer
+	cmd.Stdout = &outBuf
```

Comment:
You should probably use an io.Pipe here instead.

Reply:
yes, I think this will increase performance. I will try it out. Thanks!